internal/tiltfile/tiltfile.go

```diff
@@ -25,3 +26,34 @@ func Load(filename string) (*Tiltfile, error) {
 
 	return &Tiltfile{globals, filename, thread}, nil
 }
+
+func (tiltfile Tiltfile) GetServiceConfig(serviceName string) (*string, error) {
```

Comment:
String pointer! Exciting :)

Reply:
ya, this seems non-idiomatic. most functions like this return (string, error) and return the empty string on error